src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -467,7 +467,6 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp job stop') as c:
         c.argument('job_execution_name', help='name of the specific job execution which needs to be stopped.')
-        c.argument('execution_name_list', help='comma separated list of job execution names.')
```

Comment:
We should deprecate it first.

Reply:
@Greedygre this option was never worked end-to-end. We recently updated the stop RP endpoint to not take in the list of executions and wanted to remove this from CLI as well.

Reply:
Hi @anandanthony 

We can deprecate it an not pass through this to the RP, you can refer this PR: https://github.com/Azure/azure-cli/pull/29746